ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/ChatText.qml

```diff
@@ -45,35 +44,41 @@ Item {
             if(contentType === Constants.stickerType) return "";
             let msg = Utils.linkifyAndXSS(message);
             if(isEmoji) {
-                return Emoji.parse(msg, "72x72");
+                return Emoji.parse(msg, "72x72", "72x72");
```

Comment:
We really should put those in Constants.
```
Constants.smallEmoji: "20x20"
Constants.bigEmoji: "72x72"
```

Reply:
Constants added, but I've also removed the need for passing the source size parameter, as it was always `72x72`.